pkg/docker/docker_client.go

```diff
@@ -132,6 +146,69 @@ func NewClient(defaultHost string) (dockerClient client.CommonAPIClient, dockerH
 	return dockerClient, dockerHostInRemote, err
 }
 
+// If the DOCKER_TLS_VERIFY environment variable is set
+// this function returns HTTP client with appropriately configured TLS config.
+// Otherwise, nil is returned.
+func newHttpClient() *http.Client {
+	tlsVerifyStr, tlsVerifyChanged := os.LookupEnv("DOCKER_TLS_VERIFY")
+
+	if !tlsVerifyChanged {
+		return nil
+	}
+
+	var tlsOpts []func(*tls.Config)
+
+	tlsVerify, _ := strconv.ParseBool(tlsVerifyStr)
+	if !tlsVerify {
+		tlsOpts = append(tlsOpts, func(t *tls.Config) {
+			t.InsecureSkipVerify = true
+		})
+	}
+
+	dockerCertPath := os.Getenv("DOCKER_CERT_PATH")
```

Comment:
I'd argue if `DOCKER_CERT_PATH` isn't set, all the client certificate code below should be skipped. This requires the daemon to be configured with optional TLS verify.

Reply:
>Would delete this.

>I'd argue if DOCKER_CERT_PATH isn't set, all the client certificate code below should be skipped. This requires the daemon to be configured with optional TLS verify.

I am just trying to replicate `docker` CLI here, and this is what it does.

Reply:
The `docker` CLI uses default config path when the `DOCKER_CERT_PATH` envvar is not set. I am just trying to be compatible here.